src/Http/Exception/ForbiddenException.php

```diff
@@ -26,10 +29,10 @@ class ForbiddenException extends HttpException
      * Constructor
      *
      * @param string|null $message If no message is given 'Forbidden' will be the message
-     * @param int $code Status code, defaults to 403
+     * @param int|null $code Status code, defaults to 403
      * @param \Exception|null $previous The previous exception.
      */
-    public function __construct($message = null, $code = null, $previous = null)
+    public function __construct(?string $message = null, ?int $code = null, ?Exception $previous = null)
```

Comment:
Should this be `Throwable`? We likely didn't use it before because of PHP5 compatibility.

Reply:
Yup, it was auto added by the tool based on docblock. I'll update it to `Throwable`.